src/arch/x86_64/mm/paging.rs

```diff
@@ -616,21 +616,6 @@ pub fn get_page_table_entry<S: PageSize>(virtual_address: VirtAddr) -> Option<Pa
 	root_pagetable.get_page_table_entry(page)
 }
 
-pub fn get_physical_address<S: PageSize>(virtual_address: VirtAddr) -> PhysAddr {
```

Comment:
This fn is really not used? I could imagine it beeing helpful in debugging contexts

Reply:
It was introduced in https://github.com/hermitcore/libhermit-rs/commit/830bbc51740f97a0dca9a50c0ea30b9bc78290fc and was never used since, as far as I can tell. Do you want me to keep it for debugging purposes though?